srcpkgs/OpenRCT2/template

```diff
@@ -53,16 +56,38 @@ if [ "$CROSS_BUILD" ]; then
 	CXXFLAGS+=" -DHAVE_IMMINTRIN_H=false -DSDL_DISABLE_IMMINTRIN_H=1"
 fi
 
-pre_configure() {
-	if [ "$CROSS_BUILD" ]; then
-		sed -i -e 's#COMMAND ./openrct2#COMMAND openrct2#g' CMakeLists.txt
-	fi
-}
+if [ "$XBPS_CHECK_PKGS" ]; then
+	configure_args+=" -DWITH_TESTS=1 -DSYSTEM_GTEST=1"
+	# download replays, used in tests
+	distfiles+=" https://github.com/OpenRCT2/replays/releases/download/v${_replays_version}/replays.zip"
+	checksum+=" 43ba763ee6950a51aa96b26f3007c18b6e2ed09b6003601df6b014d5642c08d8"
+fi
 
 post_extract() {
 	_srcdir="${XBPS_SRCDISTDIR}/${pkgname}-${version}"
+	_builddir="${build_wrksrc:+$build_wrksrc/}${cmake_builddir:=build}"
 	unzip -qd data/object "${_srcdir}/objects.zip"
 	unzip -qd data/sequence "${_srcdir}/title-sequences.zip"
+	if [ "$XBPS_CHECK_PKGS" ]; then
+		mkdir -p "${_builddir}/testdata"
+		unzip -qd "${_builddir}/testdata/replays" "${_srcdir}/replays.zip"
+		# other testdata needs to be in builddir for tests to work
+		cp -a test/tests/testdata "${_builddir}/"
+		# CMakeLists seems to assume `install` is run before `check`, and so
+		# doesn't actually make the `data` symlink in time
+		ln -rs data "${_builddir}/data"
+	fi
 
 	vsed -i CMakeLists.txt -e "s; -Werror;;"
 }
+
+pre_configure() {
+	if [ "$CROSS_BUILD" ]; then
+		sed -i -e 's#COMMAND ./openrct2#COMMAND openrct2#g' CMakeLists.txt
+	fi
+}
+
+post_install() {
+	# remove useless static lib
+	rm -f "${DESTDIR}/usr/lib/libopenrct2.a"
```

Comment:
I prefer not having `-f`, so we can remove the command if they stop installing it.